solr/tests/unittests/test_solr.py

```diff
@@ -41,7 +52,21 @@ def test_cleanup_solr_request(self):
         cleaned = SolrInterface.cleanup_solr_request(payload)
         self.assertNotIn('*', cleaned['fl'])
 
-
+        
+    def test_limits(self):
+        """
+        Prevent users from getting certain data
+        """
+        db.session.add(Limits(uid='9', field='full', filter='bibstem:apj'))
+        db.session.commit()
+        assert len(db.session.query(Limits).filter_by(uid='9').all()) == 1
```

Comment:
This is only nitpicking, how about `self.assertEqual` here to be consistent with your later assertions?